hack/golangci.yaml.in

```
@@ -46,6 +46,12 @@ issues:
       text: should not be used because managedFields was removed
       path: _test.go$
 
+        # SSA Extract calls are allowed in tests.
```

Looks like it was copy-and-pasted from above? The indention is also off.

```suggestion
      # Not using emulated versions is fine for tests.
```